### DISCUSSION TITLE
My wallet not synchronized!!!

### DISCUSSION THREAD

**Discussion by Stalker0691** [Chia-Network/chia-blockchain#6961] [Support]
my wallet status is not synchronized, I ask you to help synchronize the wallet and get my chia coins. Full Node Status - SYNCHRONIZED!!!

**Comment by loppefaaret** [comment]
does it never move ? is there any ERROR or WARNING in `debug.log` from the wallet ?

**Comment by Stalker0691** [comment]
Please view the log file, I also send a screenshot of the wallet and the full node, the wallet does not move from this value for almost a month … Help me set up a farm, it has been working for more than a month, about 8 terabytes of disk space, no earnings, 0 in my wallet ...
 
 
Karen Manukyan 
Technical Support
X-Art LLC
Yerevan, Armenia 0002
Office: Saryan 4/6 | Store: Saryan 4/55
Tel:  +37410446677  | Mob:  +37499530972
***@***.***  |  www.x-art.am

**Reply by loppefaaret** [reply]
WINDOWS WALLET DATABASE RESET

Close down the Chia app, and open up a PowerShell window, and execute the below command:
```
mv ~\.chia\mainnet\wallet\db\block* ~\Desktop\
```
Then try opening up the Chia app again
if it worked, you can remove the files we put on the Desktop!

LINUX WALLET DATABASE RESET

Close down the Chia, and open up a terminal, and execute the below command:
```
mv ~/.chia/mainnet/wallet/db/block* ~/Desktop/
```
Then try opening up the Chia again
if it worked, you can remove the files we put on the Desktop!

**Comment by Stalker0691** [comment]
I did as you described, now I have a complete reset of the wallet, how long should I wait for synchronization?
Thank you.
 
 
Karen Manukyan 
Technical Support
X-Art LLC
Yerevan, Armenia 0002
Office: Saryan 4/6 | Store: Saryan 4/55
Tel:  +37410446677  | Mob:  +37499530972
***@***.***  |  www.x-art.am

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
